src/java/org/apache/cassandra/tools/nodetool/Status.java

```diff
@@ -48,6 +56,21 @@ public class Status extends NodeToolCmd
     @Option(title = "resolve_ip", name = {"-r", "--resolve-ip"}, description = "Show node domain names instead of IPs")
     private boolean resolveIp = false;
 
+    @Option(title = "sort",
+            name = {"-s", "--sort"},
+            description = "Sort by one of 'ip', 'host', 'load', 'owns', 'id', 'rack' or 'state'. " +
+                          "Default ordering is ascending for 'ip', 'host', 'id', 'token', 'rack' and descending for 'load', 'owns', 'state'. " +
+                          "Sorting by token is possible only when cluster does not use vnodes. When using vnodes, default " +
+                          "sorting is by id otherwise by token.",
+            allowedValues = {"ip", "host", "load", "owns", "id", "rack", "state", "none"})
```

Comment:
If we are also including token do we not have to include token in the allowed values?